ibis/backends/bigquery/client.py

```diff
@@ -194,7 +194,53 @@ def bq_param_date(_: dt.Date, value, name):
 
 
 class BigQueryTable(ops.DatabaseTable):
-    pass
+    @property
+    def bq_table(self):
```

Comment:
Not sure that we want to be exposing these classes directly. It's useful, but also more tightly couples Ibis to the BigQuery client library.

Reply:
Agreed — i made it an internal property `_bq_table` so that we can cache it and access it repeatedly.